### PR TITLE
Update placeholder in plist file to use consistent pattern for placeholder

### PR DESCRIPTION
**Pull request by haishengwu-okta**

### CODE REVIEW

**Review by IldarAbdullin-okta — APPROVED** [review]
LGTM!

**Comment by haishengwu-okta** [comment]
Looks like the custom scheme uri doesn't have to match okta tenant.
How about hard code to `com.okta.sample-app` cross the board?

**Comment by IldarAbdullin-okta** [comment]
> Looks like the custom scheme uri doesn't have to match okta tenant.
> How about hard code to `com.okta.sample-app` cross the board?

Then you will need to do: <string>com.okta.sample-app:/callback</string> in Okta.plist for consistency.
I think {customUriScheme} template is fine

File: browser-sign-in-and-biometric-storage/OktaBrowserSignIn/Okta.plist [hunk 1]
```diff
@@ -5,7 +5,7 @@
 	<key>scopes</key>
 	<string>openid profile offline_access</string>
 	<key>redirectUri</key>
-	<string>{customUriScheme}:/callback</string>
+	<string>{customUriScheme}:/loginCallback</string>
```
Comment (IldarAbdullin-okta) [inline browser-sign-in-and-biometric-storage/OktaBrowserSignIn/Okta.plist:8]: 👍

Comment (haishengwu-okta) [reply browser-sign-in-and-biometric-storage/OktaBrowserSignIn/Okta.plist:8]: 🙌  Just to make consistent between iOS and react-native sample.

**Pull request merged**